skythought/tools/util/task_handlers.py

```diff
@@ -105,10 +105,27 @@ class AIMETaskHandler(MathTaskHandler):
     def __init__(self):
         self.dataset = "AI-MO/aimo-validation-aime"
     
+    @staticmethod
+    def generate_prompt(prompt, model):
+        if "NovaSky-AI/Sky-T1-32B-Preview" in model:
```

Comment:
can this be written as a map for better extensiblity? 

https://github.com/NovaSky-AI/SkyThought/blob/main/skythought/tools/util/model_utils.py#L55